### PR TITLE
Fix icon on the Polkadot Block Explorer Menu item #3974

### PR DESCRIPTION
**Pull request by oleksanderkorn**
- #3974
image
image

### CODE REVIEW

**Review by traumschule — APPROVED** [review]
:ok_hand:

**Comment by traumschule** [comment]
On a closer look
1. The link icon should be aligned right inside the bar.
2. Tooltip is currently not implemented.

**Comment by traumschule** [comment]
@thesan i suggest to merge this good fix and let @vrrayz do the fine-tuning.

File: packages/ui/src/common/components/page/Sidebar/NavigationLink.tsx [hunk 1]
```diff
@@ -119,7 +119,21 @@ const NavigationItemLinkChildren = styled.div`
   ${Overflow.FullDots};
 
   svg {
-    width: 20px;
+    width: 16px;
+    height: 16px;
+
+    path {
+      fill: ${Colors.Black[400]}!important;
```
Comment (thesan) [inline packages/ui/src/common/components/page/Sidebar/NavigationLink.tsx:126]: @oleksanderkorn does [`currentcolor`](https://css-tricks.com/currentcolor/) so the hover could be removed ?
```suggestion
      fill: currentcolor !important;
```

**Review by thesan — APPROVED** [review]

**Pull request merged**